queue-system.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+require_once 'env.class.php';
+$__DotEnvironment = new DotEnvironment(realpath("./.env"));
+
+// Set up a database connection
+$db = new PDO('mysql:host="'.$_ENV['DB_HOST'].'";dbname="'.$_ENV['DB_NAME'].'"', '"'.$_ENV['DB_USER'].'"', '"'.$_ENV['DB_PASSWORD'].'"');
+
+// Define a function to add a new email to the queue
+function add_email_to_queue($to, $subject, $message) {
```

Comment:
This does not use proper OOP concepts. The queue should be a class that can be reused anywhere. What you have is a file containing several functions.

Reply:
checked